README.md

```diff
@@ -9,8 +9,9 @@ Data for Democracy's mission is to be an inclusive community for data scientists
 * Read about our [current projects](#current-projects).  
 * If you see a project you are interested in, join the project slack channel, introduce yourself to the community and the project lead(s).  
 * Visit project readme for specific instructions on who to contact or how to get started.
-* Not sure which project you want to join? Introduce yourself in #general. Talk to a few project leads, let them know what you are interested in. Project leads can help you determine if a project is a good fit for you.  
+* Not sure which project you want to join? Introduce yourself in #general. Talk to a few project leads, let them know what you are interested in. Project leads can help you determine if a project is a good fit for you.
 * Have an idea for a new project? Join [#project-ideas](https://datafordemocracy.slack.com/messages/project-ideas/) to start a discussion on a new project.
+* Looking for a certain type of task, in any project? Check the [task digest](#task-digest).
```

Comment:
needs include full header `#task-digest-non-exhaustive` to be able to jump to heading